.github/workflows/calypso-live.yml

```diff
@@ -12,18 +12,38 @@ jobs:
     if: github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
     timeout-minutes: 10
     steps:
+      - name: Check if comment exists
+        id: has_comment
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          result-encoding: string
+          script: |
+            const comments = await github.rest.issues.listComments( {
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            } );
+            // For debugging, if needed!
+            console.log( comments );
+            const existingComment = comments.data.find( comment => comment.body.includes( "calypso-live-watermark:apr@v1" ) );
+            const hasComment = existingComment ? "true" : "false";
+            console.log( `Has existing comment: ${ hasComment }` );
```

Comment:
Should we clean those `console.log()` calls up before shipping?

Reply:
I wanted to keep them because they provide helpful info when a build fails. It's only in a dev build log, so the only reason you'd check it is if you wanted to get information about what the build was doing. So I think it makes sense to keep them here. (Otherwise, debugging issues gets harder because you have to spin up a new branch with your own logs)